src/source/Signaling/LwsApiCalls.c

```diff
@@ -2393,6 +2393,13 @@ PVOID receiveLwsMessageWrapper(PVOID args)
     }
     // Calling client receive message callback if specified
     if (pSignalingClient->signalingClientCallbacks.messageReceivedFn != NULL) {
+        if (messageType == SIGNALING_MESSAGE_TYPE_OFFER) {
```

Comment:
Why are we adding this section here? Cant we reuse: `pSignalingClient->offerReceivedTime = GETTIME();` which is on line 2376? Same with answer.

Reply:
Removed it. I didn't notice that earlier because it was not present in the SignalingClientStats that I'm using later